web/themes/joinup/prototype/sass/components/html/listing-item--video-col-12.html

```diff
@@ -0,0 +1,38 @@
+<div style="background-color: #e9edee;">
+  <div class="listing listing--grid mdl-grid">
+    <div class="listing__item listing__item--tile mdl-cell mdl-cell--12-col">
+      <div class="listing__card listing__card--inline-container mdl-card">
+        <div class="listing__image listing__image--default">
+          <span class="icon icon--play"></span>
```

Comment:
Could you please try to make the area with the icon clickable to have the same behavior as other icons? Maybe putting the classes to an a tag could be ok?